### ISSUE TITLE
Issues with bundle creation targeting Windows 10 /Hyper-V

### ISSUE THREAD

**Issue by andrejpetras** [code-ready/snc#78]
Hi,

I try to build my own Hyper-V bundle and test it on the windows 10.
I get an error:  `id_rsa_crc: Access is denied`
Do I need to setup any special rights on windows 10?

```                                                                   
λ  .\crc.exe version                                                                        
version: 0.89.1-alpha-4.1.6+3946ae0                                                         
C:\work\programs\crc            
λ  .\crc.exe start --log-level debug -b crc_hyperv_4.1.6.crcbundle                                                                                                             
INFO Checking if oc binary is cached                                                                                                                                           
DEBU oc binary already cached                                                                                                                                                  
INFO Checking if CRC bundle is cached in '$HOME/.crc'                                                                                                                          
INFO Check Windows 10 release                                                                                                                                                  
INFO Hyper-V installed and operational                                                                                                                                         
INFO Is user a member of the Hyper-V Administrators group                                                                                                                      
INFO Does the Hyper-V virtual switch exist                                                                                                                                     
INFO Extracting the crc_hyperv_4.1.6.crcbundle Bundle tarball ...                                                                                                              
ERRO Error occurred: Error to get bundle Metadata Error during extraction : open C:\Users\apetras\.crc\cache\crc_hyperv_2019-08-07\id_rsa_crc: Access is denied.               
C:\work\programs\crc                                                                                                                                                                                                            
```

**Comment by andrejpetras** [comment]
On RHEL8 I need to change this lines in the createdisk.sh script
```
   sudo cp /var/lib/libvirt/images/${VM_PREFIX}-master-0 $destDir
    sudo cp /var/lib/libvirt/images/${VM_PREFIX}-base $destDir
```
to

```
   sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${VM_PREFIX}-master-0 $destDir
    sudo cp /var/lib/libvirt/openshift-images/${VM_PREFIX}/${VM_PREFIX}-base $destDir
```

**Comment by praveenkumar** [comment]
@andrejpetras yes, this is something recently changed in the libvirt provider side. we will need to update the master for this change.

**Comment by praveenkumar** [comment]
Fixed by #93
